app/views/audits/audits/_content_items.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= render 'audits/common/counter', locals: { content_itemsitems: content_items } %>
```

Comment:
This is still working because the partial is falling back to using the `content_items` helper method because we've misspelled the local variable. That would be revealed if we only gave the partial the data it needs.

Reply:
Sorry I tried to finish it in a rush before leaving. Will do it properly once I get back.